pluginloader/__init__.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python

```

Comment:
This is not something we run stand-alone, why have it there at all? If it was for stand-alone use, it should say `python2` to conform to PEP 394.